packages/dds/merge-tree/package.json

```diff
@@ -56,6 +56,7 @@
     "@fluidframework/common-definitions": "^0.19.1",
     "@fluidframework/common-utils": "^0.27.0",
     "@fluidframework/container-definitions": "^0.35.0",
+    "@fluidframework/container-utils": "^0.35.0",
```

Comment:
ok. it looks like this is allowed, but i'm not sure it should be. i'd prefer to move the error to telemetry utils, which is a bigger change